fastlane/lib/fastlane/plugins/templates/plugin.rb.erb

```diff
@@ -0,0 +1,16 @@
+require '<%= require_path %>/version'
+
+module Fastlane
+  module <%= plugin_name.capitalize %>
```

Comment:
Ah, that doesn't work, this should be `plugin_name.fastlane_class`, so it works with `_` too 👍